README.md

```diff
@@ -34,9 +34,11 @@ Then, use the factory that is provided to create the client:
 ```php
 $client = JoliCode\Slack\ClientFactory::create($yourToken);
 
-$client->apiTest(); // $client contains all the methods to interact with the API
+$client->usersList(); // $client contains all the methods to interact with the API
```

Comment:
You should do something with the returned value. The main README should show what is returned (a nice object), because users never read / go to another file :grimacing: